assets/components/directDebit/directDebitActions.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+// ----- Types ----- //
+
+export type Action =
+  | { type: 'DIRECT_DEBIT_POP_UP_OPEN' }
+  | { type: 'DIRECT_DEBIT_POP_UP_CLOSE' }
+  | { type: 'DIRECT_DEBIT_UPDATE_SORT_CODE', sortCode: string }
+  | { type: 'DIRECT_DEBIT_UPDATE_ACCOUNT_NUMBER', accountNumber: string }
+  | { type: 'DIRECT_DEBIT_UPDATE_ACCOUNT_HOLDER_NAME', accountHolderName: string }
+  | { type: 'DIRECT_DEBIT_UPDATE_ACCOUNT_HOLDER_CONFIRMATION', accountHolderConfirmation: boolean };
+
+
+// ----- Actions ----- //
+
+const openDirectDebitPopUp = (): Action => ({ type: 'DIRECT_DEBIT_POP_UP_OPEN' });
+
+const closeDirectDebitPopUp = (): Action => ({ type: 'DIRECT_DEBIT_POP_UP_CLOSE' });
```

Comment:
Could these be merged as `DIRECT_DEBIT_POP_UP_OPEN`, which takes a boolean? Would that be more idiomatic? I'm not sure, it seems reasonable either way. However, it would save an extra action and an extra `case` in the reducer.

Reply:
👍